src/packagedcode/readme.py

```diff
@@ -0,0 +1,103 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# ScanCode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/scancode-toolkit for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+import logging
+
+import attr
+from commoncode import filetype
+from commoncode import fileutils
+
+from packagedcode import models
+
+"""
+Handle README.* package metadata
+"""
+
+TRACE = False
+
+logger = logging.getLogger(__name__)
+
+if TRACE:
+    import sys
+    logging.basicConfig(level=logging.DEBUG, stream=sys.stdout)
+    logger.setLevel(logging.DEBUG)
+
+
+@attr.s()
+class ReadmePackage(models.Package):
+    metafiles = (
+        'README.android',
+        'README.chromium',
+        'README.facebook',
+        'README.google',
+        'README.thirdparty',
+    )
+    default_type = 'readme'
+
+    @classmethod
+    def recognize(cls, location):
+        yield parse(location)
+
+    @classmethod
+    def get_package_root(cls, manifest_resource, codebase):
+        return manifest_resource.parent(codebase)
+
+    def compute_normalized_license(self):
+        return models.compute_normalized_license(self.declared_license)
```

Comment:
`compute_normalized_license()` does not seem to work for strings like `BSD` and `Apache`. I am guessing this is by design to limit false-positives. However, these are primarily how licenses are listed in `README.*` files.

@pombredanne What would be the best way to do detection on the `declared_license` field here?

Reply:
@majurg we would need a mapping and override compute_normalized_license() here